### PR TITLE
Rpi flow

### PR DESCRIPTION
**Pull request by dilin-MS**

### CODE REVIEW

File: resources/templates/platformlist.json [hunk 1]
```diff
@@ -11,8 +11,8 @@
             "description": "Project based on Linux(Yocto/Ubuntu/Debian/...) Platform."
         },
         {
-            "name": "$(issue-opened) My device is not in the list...",
-            "id": "no_device",
+            "name": "$(issue-opened) My wanted platform is not in the list...",
```
Comment (lirenhe) [inline resources/templates/platformlist.json]: let's use device for now

File: src/Models/IoTWorkbenchProjectBase.ts [hunk 2]
```diff
@@ -1,12 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+import {RetryOperation} from 'azure-iot-common';
```
Comment (lirenhe) [inline src/Models/IoTWorkbenchProjectBase.ts]: do we need this?

File: src/Models/IoTWorkbenchProjectBase.ts [hunk 3]
```diff
@@ -270,4 +274,21 @@ export abstract class IoTWorkbenchProjectBase {
   }
 
   abstract async handleLoadFailure(): Promise<boolean>;
+
+  static async generateIotWorkbenchProjectFile(
```
Comment (lirenhe) [inline src/Models/IoTWorkbenchProjectBase.ts]: no need to be static

File: src/Models/IoTWorkbenchProjectBase.ts (hunk 2 above)
Comment (dilin-MS) [reply src/Models/IoTWorkbenchProjectBase.ts]: No.

File: resources/templates/platformlist.json (hunk 1 above)
Comment (dilin-MS) [reply resources/templates/platformlist.json]: Fixed.

File: src/Models/IoTWorkbenchProjectBase.ts (hunk 3 above)
Comment (dilin-MS) [reply src/Models/IoTWorkbenchProjectBase.ts]: In order to be called in `RaspberryPiDevice.ts`:  
`await IoTWorkbenchProjectBase.generateIotWorkbenchProjectFile(...);`

**Pull request merged**